interp/constrextern.ml

```diff
@@ -63,6 +63,31 @@ let print_universes = Detyping.print_universes
 (* This suppresses printing of primitive tokens (e.g. numeral) and notations *)
 let print_no_symbol = ref false
 
+(* This tells to skip types if a variable has this type by default *)
+let print_use_implicit_types = ref true
+
+let () =
+  let open Goptions in
+  declare_bool_option
```

Comment:
declare_bool_option_and_ref is preferred these days

Reply:
I did not know. Thanks, that's indeed better.

Is there some ongoing project to replace all `declare_bool_option` with `declare_bool_option_and_ref`? If not, I can do a couple of them...

Reply:
@SkySkimmer: actually, how do implement things like `let without_symbols f = Flags.with_option print_no_symbol f` if `print_no_symbol` is defined using `declare_bool_option_and_ref`?

Reply:
I guess that type of usage is one reason why _and_ref isn't used everywhere.

Reply:
The idea of `declare_bool_option_and_ref` is nice. It can certainly be extended to also support a `without` scheme.